packages/trader/src/AppV2/app.tsx

```diff
@@ -41,10 +41,14 @@ const App = ({ passthrough }: Apptypes) => {
     }, []);
 
     React.useEffect(() => {
-        if (!window.location.pathname.startsWith('/contract')) {
+        if (
+            !window.location.pathname.startsWith('/contract') &&
```

Comment:
If you want to send only when it's trading page, maybe we can rework check like this:
`window.location.pathname === routes.trade`
?